src/app/geo/map/leaflet-map/leaflet-config.ts

```diff
@@ -5,26 +5,28 @@ export class LeafletConfig {
   static INIT_ZOOM = 14;
   static CONTROL_ZOOM_POSITION: ZoomOptions = { position: 'topright' };
   private static MARKERS_PATH = '../assets/leaflet-icons/';
```

Comment:
I think the warning messages were due to the map tiles not the marker images. 
The trailing in line 15 must go out of course!